sdk/storage/Azure.Storage.Common/swagger/Generator/src/generator.ts

```diff
@@ -557,9 +554,9 @@ function generateOperation(w: IndentWriter, serviceModel: IServiceModel, group:
                     w.line(`if (${responseName}.Headers.TryGetValue("${header.name}", out ${headerName}))`);
                     w.scope('{', '}', () => {
                         w.write(`${valueName}.${naming.pascalCase(header.clientName)} = `);
-                        if (isPrimitiveType(header.model) && header.model.collectionFormat === `multi`) {
+                        if (isPrimitiveType(header.model) && header.model.collectionFormat === `csv`) {

```

Comment:
Is this related to the PR?

Reply:
Yes - the part where we change from `multi` to `csv` for header params.